src/client/components/common/Combobox.tsx

```diff
@@ -280,12 +279,9 @@ export const Combobox: FC<ComboboxProps> = ({
                 }
                 {...getInputProps({
                   ...props,
+                  placeholder: 'Select an option',
                   ref: inputRef,
                   onFocus: () => {
-                    // reset field for new search
```

Comment:
I've discussed this with Natalie already, and it seems that she wants this behaviour as it might be closer to what the user expects with an input field. If users complain about this we can always revert back to the original clear input on focus implementation.